test/bats-unit-test/run-bats-Unit-tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+sudo apt-get install -y bats &> /dev/null
```

Comment:
Is this just covering both package managers and swallowing errors? 

How does one determine if there's a genuine problem getting the bats package?

Reply:
good point. I will address by generate a log.

Reply:
I addressed this by checking if the package existed and if not installing the packages without redirecting output